proxy/src/control/discovery.rs

```diff
@@ -61,7 +60,6 @@ pub struct Background {
 
 /// A future returned from `Background::work()`, doing the work of talking to
 /// the controller destination API.
-// TODO: debug impl
```

Comment:
Is there a reason we no longer want an impl of `Debug` for this struct?

Reply:
I don't think it's a helpful todo. Also, we clearly don't need it to impl Debug for any concrete reason (and this file is already big enough).